Adafruit_TLC5947.cpp

```diff
@@ -92,6 +92,22 @@ void Adafruit_TLC5947::setPWM(uint16_t chan, uint16_t pwm) {
   pwmbuffer[chan] = pwm;
 }
 
+/*!
+ *    @brief  Get the PWM value for channel.
+ *    @param  chan
+ *            channel number ([0 - 23] on each board, so channel 2 for second
+ * board will be 25)
+ *    @return PWM value ([0 - 4095]) for valid channels, 0 for non-existing
+ * channels.
+ */
+uint16_t Adafruit_TLC5947::getPWM(uint16_t chan) {
+  if (chan >= 24 * numdrivers) {
+    return 0;
```

Comment:
`0` is a potentially valid return, so this would mask trying to access an invalid channel. Could maybe return null?

Reply:
Yes, but returning numeric `0` for a non-existing channel seems the "least wrong" to me: If a channel does not exist it cannot output PWM. Thus, the PWM value returned by `getPWM()` for any non-valid channel always signifies "off" (i.e. `0`). 

Moreover, using `setPWM()` on non-existing channels is also ignored _silently_. Returning 0 from `getPWM` appears the closest possible equivalent of this to me. 

Returning `NULL`  from `getPWM()` might be dangerous, because depending on compiler and exact version of C/C++, it can be defined differently, either as `#define NULL 0` or `#define NULL nullptr` (or maybe even somthing else?). The former is the same as directly returning numerical `0`, the latter will probably cause a compile time error.

As an alternative, one could return an "obviously" non-valid PWM setting for non-existing channels, something like `4096 `or `UINT16_MAX `(i.e. `65535`), so there would be no ambiguities. However, I doubt that would make debugging of user code any easier. If a user tries to get the PWM value from a non-existing channel and do something useful with this, there are serious problems with the code in any case... :-)

Thus, I would rather prefer `0` as the least-surprising return value for non-existing channels, as described above.

Reply:
I am with agrommek here, as `setPWM()` gives no feedback, also `getPWM()` can have none.

Could this PR then be merged?

On a low memory device (ATtiny) this would be more than welcome.